tests/pkey/include.am

```diff
@@ -2,6 +2,7 @@
 # included from top level Makefile.am
 # ALl path should be given relative to root directory
 
+dist_noinst_SCRIPTS+=tests/pkey/ecparam-test.sh
```

Comment:
This file appears to be missing: `make[1]: *** No rule to make target `tests/pkey/ecparam-test.sh', needed by `all-am'.  Stop.`

Reply:
Thanks, added it in